src/transaction.js

```diff
@@ -768,7 +768,13 @@ const transaction = {
         if (response.success) {
           resolve(response);
         } else {
-          reject(response.message);
+          let message = '';
+          if (response.return_code === 'stratum_timeout') {
+            message = 'All transactions need to resolve proof-of-work as an anti spam mechanism. Currently Hathor Network provides their full nodes to execute this task for free. However they may be fully loaded right now.\n\nThe bigger the transactions, the harder it is to resolve the proof of work.\n\nPlease, wait a few moments and try again soon and, if you can split in more transactions it will be easier to send them.';
```

Comment:
I'd only change to: "and try again soon. If you can split in two or more smaller transactions, it will be easier to send them. Notice that what matters most is the number of inputs (instead of the amount being transferred)."

Is it confusing?

Reply:
- We don't seem to say anywhere "Timeout solving transaction's proof-of-work". Or do we say it on the wallet modal? This is the most important information.

- Should we use "Hathor Network" or "Hathor Labs"?

- @msbrogli I think your suggestion is a bit confusing. Even if they have this information, there's not much they can do. People don't deal with the number of inputs, just the amount. Unless they manually select the inputs, which is very unlikely.

- My suggestion:
```
Timeout solving transaction's proof-of-work.

All transactions need to solve a proof-of-work as an anti spam mechanism. Currently, Hathor Labs provides this service for free, but their servers may be fully loaded right now.

The bigger the transactions, the harder it is to solve the proof-of-work. Please, wait a few moments and try again soon. If you can split your transaction into smaller ones it might be easier to send them.
```

Reply:
I agree with @obiyankenobi that the inputs information is confusing, I've updated to his suggestion.

![image](https://user-images.githubusercontent.com/3298774/80763028-0522e080-8b14-11ea-9623-90096cc5c5e9.png)

Reply:
I think the last part "If you split your transaction into smaller ones it might be easier to send them" may mislead to the understanding that the amount of the transaction is important. On the other hand, I also agree that talking about inputs is more confusing.

Ok. I agree that we can move forward.